server/controllers/list.js

```diff
@@ -569,29 +593,76 @@ const changeToOwner = (req, resp) => {
     });
 };
 
-const changeToMember = (req, resp) => {
+const removeOwnerRole = (req, resp) => {
   const { id: listId } = req.params;
   const { userId } = req.body;
+
   const {
     user: { _id: ownerId }
   } = req;
 
-  List.findOne({ _id: listId, ownerIds: { $all: [ownerId, userId] } })
+  List.findOne({ _id: listId, ownerIds: ownerId })
     .populate('cohortId', 'memberIds ownerIds')
```

Comment:
is this populate necessary here? I can't see an usage of cohortId? The same in addOwnerRoleController